### PR TITLE
Document callbacks for Forward

### PR DESCRIPTION
**Pull request by ghost**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-forward-callbacks.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/verbs/forward.md [hunk 1]
```diff
@@ -14,7 +14,11 @@ Forwards an unanswered incoming call to another number.
 
 ### Callbacks Received
 
-None
+| Callbacks                                 | Can reply with more BXML |
+|:------------------------------------------|:-------------------------|
+| [Disconnect](../callbacks/disconnect.md)  | No                       |
+
+**NOTE:** You will not receive any callbacks for the forwarded leg of the call (i.e. Answer).
```
Comment (nashley) [inline voice/bxml/verbs/forward.md]:
```suggestion
**NOTE:** You will not receive any callbacks for the forwarded leg of the call (e.g., Answer). 
```
You probably already know this, but `i.e.` roughly translates as "that is"; `e.g.` as "for example"; and `viz.` as "most notably".
We probably want to use `e.g.` or perhaps `viz.` to indicate that, while other callbacks exist, they're not possible from a user perspective (in the case of BXML callbacks) and not technically supported (in the case of `answerCallback`).

Comment (vsmith-bw) [reply voice/bxml/verbs/forward.md]: Yep; he's right about the latin. It is clear otherwise.

**Review by vsmith-bw — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-forward-callbacks.s3-website-us-east-1.amazonaws.com/

**Review by scapps-bw — CHANGES_REQUESTED** [review]
Is this Disconnect from the Leg A or Leg B? It seems to be Leg A given the Note, in which case why are we saying the Forward is generating the Disconnect? Not sure I understand. I would find this confusing as a Bandwidth customer.

**Comment by ghost** [comment]
Do customers know A-leg vs B-leg?  I'm trying to say you won't see any events for the B-leg, but you'll receive a disconnect for the A-leg.

Maybe I should take the callback table out.  No other verbs list the disconnect as a callback for the verb, because it's not really from the verb.

**Comment by scapps-bw** [comment]
Yeah, that's what I am feeling, that last part you said. The Disconnect is coming from the original call, not the verb, so shouldn't be listed there, IMO.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-forward-callbacks.s3-website-us-east-1.amazonaws.com/

**Review by scapps-bw — APPROVED** [review]

**Pull request merged**